packages/cli/src/cli.ts

```diff
@@ -49,6 +53,7 @@ Commands:
 
 Options:
   --version, -v   Print the version of the command-line tool
+  --prefer-enterprise, -p Prefer AG Grid enterprise imports
```

Comment:
We may want to adjust this given we may need more complex inputs from the user for the next code mods. 

Happy to merge this for now so we have the code in place.